packages/wasm-dpp/test/integration/stateTransition/StateTransitionFacade.spec.js

```diff
@@ -28,85 +19,100 @@ describe('StateTransitionFacade', () => {
   let stateRepositoryMock;
   let dataContract;
   let identityPublicKey;
+  let identity;
+
+  let DashPlatformProtocol;
+  let DataContractFactory;
+  let DataContractValidator;
+  let DataContractCreateTransition;
+  let Identity;
+  let ValidationResult;
+  let IdentityPublicKey;
+  let DocumentFactory;
+  let DocumentValidator;
+  let ProtocolVersionValidator;
+  let IdentityCreateTransition;
+  let UnsupportedProtocolVersionError;
+  let InvalidStateTransitionSignatureError;
+  let DataContractAlreadyPresentError;
+  let BalanceIsNotEnoughError;
+
+  before(async () => {
+    ({
+      DashPlatformProtocol,
+      ValidationResult,
+      DataContractCreateTransition,
+      Identity,
+      DataContractValidator,
+      DataContractFactory,
+      IdentityPublicKey,
+      DocumentFactory,
+      DocumentValidator,
+      ProtocolVersionValidator,
+      IdentityCreateTransition,
+      UnsupportedProtocolVersionError,
+      InvalidStateTransitionSignatureError,
+      DataContractAlreadyPresentError,
+      BalanceIsNotEnoughError,
+    } = await loadWasmDpp());
+  });
 
   beforeEach(async function beforeEach() {
     const privateKeyModel = new PrivateKey();
-    const privateKey = privateKeyModel.toWIF();
+    const privateKey = privateKeyModel.toBuffer();
     const publicKey = privateKeyModel.toPublicKey().toBuffer();
     const publicKeyId = 1;
 
-    identityPublicKey = new IdentityPublicKey()
-      .setId(publicKeyId)
-      .setType(IdentityPublicKey.TYPES.ECDSA_SECP256K1)
-      .setData(publicKey)
-      .setPurpose(IdentityPublicKey.PURPOSES.AUTHENTICATION)
-      .setSecurityLevel(IdentityPublicKey.PURPOSES.MASTER);
+    // TODO: check security level. It says it needs HIGH instead of MASTER
```

Comment:
I think you've just fixed this comment with the code below 😉

Reply:
Yeah, the thing is that original test was providing MASTER, but rs-dpp won't accept that. 
I wonder if this is a bug or intentional behaviour.

Reply:
I think this is the intended behavior. The old test was working on mocks